### ISSUE TITLE
Feature request for academic style resume

### ISSUE THREAD

**Issue by liutiming** [rstudio/pagedown#108]
Thanks for this awesome package!

I am wondering if there is a template available for a more traditionally styled cv like [this](http://svmiller.com/blog/2016/03/svm-r-markdown-cv/). Currently the linked template use `\hfill` to format the date but I feel a more organised way maybe to have date as a field and automatically align them to the right of the page during knitting.

**Comment by yihui** [comment]
This doesn't seem like a super challenging CSS exercise. Anyone please feel free to submit a PR :)

**Comment by cderv** [comment]
@yihui I don't know if this is an option for pagedown but it may be something interesting for [tidydevday](https://github.com/tidyverse/dev-day-2019)
Anyway, I put the label help wanted to show someone could chime in.

**Comment by yihui** [comment]
Yes, that is a great idea. We could also make a virtual R Markdown dev day. There are so many issues that we need help with :)

**Comment by rbcavanaugh** [comment]
This is an old thread, but....I have been using html_resume with @nstrayer 's datadrivenCV package and needed to generate a plain/boring/academic version to use in cases where I didn't think the fancy {pagedown} resume would be well received. 

I'm a relative CSS novice, but I some hackish modifications of the pagedown/datadrivenCV CSS files have gotten me close (enough) to some of the other Rmarkdown CVs out there (e.g., http://svmiller.com/blog/2016/03/svm-r-markdown-cv/). 

However, I can't for the life of me figure out how pagedown knows to take a hyphen "-" and split the dates. My workaround for this is just to use an emdash, but that's obviously not ideal. 

Any suggestions on how to modify this? 

Here's a reprex/gist of the CSS/pagedown resume template file (note that hyphens have been swapped for emdash for date ranges): https://gist.github.com/rbcavanaugh/f3a793ebcd45bde04e862569f5ce681b

Here's a screenshot of the output
<img width="1023" alt="Screen Shot 2022-02-22 at 10 11 53 PM" src="https://user-images.githubusercontent.com/46727239/155257301-e1e8f3b2-63c5-43fa-b785-01b1a83025a5.png">

Thanks for your work on the package...I really enjoy not having to fuss with formatting a word document for my CV.

**Comment by cderv** [comment]
> However, I can't for the life of me figure out how pagedown knows to take a hyphen "-" and split the dates.

About this, this is done in JS in the `resume.html` template (line 107)
https://github.com/rstudio/pagedown/blob/7ed2764e1ed679025ee5af06a39a95cf53466955/inst/resources/html/resume.html#L95-L112

You're current trick seems ok to me if you don't want this behavior.
